### ISSUE TITLE
Make emoticon bigger if it's standalone

### ISSUE THREAD

**Issue by longnd** [overtake/telegram#419]
@overtake This is a suggestion to improve the user experience: sometime use just send emoticons without text. Please make it bigger to express the motion.
Let's take skype as an example: if message contains text & emoticon, the size of emoticon is small as usual. But if users send just emotions, it appears bigger.
Take a look at this screenshot
<img width="633" alt="screen shot 2017-03-23 at 10 15 33 am" src="https://cloud.githubusercontent.com/assets/1896814/24230449/bb33ed92-0fb1-11e7-86c1-37c644eb8e50.png">

**Comment by z3t0** [comment]
Seems like a solid idea

**Comment by Arvacon** [comment]
I agree, can you add a setting that increases the size of the emoticons, like the number for the font size at the mobile app?
